CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   ([#672](https://github.com/nix-rust/nix/pull/672))
 - Added protocol families in `AddressFamily` enum.
   ([#647](https://github.com/nix-rust/nix/pull/647))
+- Added support for OpenBSD.
```

Comment:
We didn't "add support". OpenBSD cannot be tested in CI yet, so we haven't added support here, but this PR does do is fix compilation and tests for OpenBSD targets.